pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -620,6 +628,37 @@ func makeHttpHealthCheck(name, path string, port int32) *compute.HttpHealthCheck
 	}
 }
 
+// mergeHttpHealthChecks reconciles HttpHealthCheck configures to be no smaller
+// than the default values.
+// E.g. old health check interval is 2s, new default is 8.
+// The HC interval will be reconciled to 8 seconds.
+// If the existing health check is larger than the default interval,
+// the configuration will be kept.
+func mergeHttpHealthChecks(hc, newHC *compute.HttpHealthCheck) *compute.HttpHealthCheck {
```

Comment:
Fix this in a follow on in master. (remove the return value)

We should not be both modifying stuff in place and returning the object, which implies the incoming parameters are not modified.

Reply:
Fixed in https://github.com/kubernetes/kubernetes/pull/70384.